frontend/src/components/elements/PlotlyChart/CustomTheme.tsx

```diff
@@ -462,17 +428,25 @@ export function applyStreamlitThemeTemplateData(
   if (data !== undefined) {
     data.bar.forEach((entry: any) => {
       if (entry.marker !== undefined && entry.marker.line !== undefined) {
-        entry.marker.line = assign(entry.marker.line, {
+        merge(entry.marker.line, {
```

Comment:
I think this should be removed because  color is always defined and so this really does nothing...